jlm/llvm/opt/alias-analyses/MemoryStateEncoder.cpp

```diff
@@ -872,6 +1088,10 @@ MemoryStateEncoder::EncodeGammaEntry(rvsdg::GammaNode & gammaNode)
   auto & stateMap = Context_->GetRegionalizedStateMap();
   auto memoryNodes = Context_->GetModRefSummary().GetGammaEntryNodes(gammaNode);
 
+  // Count the memory state arguments once per subregion
+  for (size_t i = 0; i < gammaNode.nsubregions(); i++)
```

Comment:
for (auto & subregion : gammaNode.Subregions())

Reply:
Hahah, yes, I should have remembered from the last two times you pointed this out :)